rocprim/include/rocprim/intrinsics/warp.hpp

```diff
@@ -47,8 +47,13 @@ ROCPRIM_DEVICE inline
 unsigned int masked_bit_count(unsigned long long x, unsigned int add = 0)
 {
     int c;
+    #ifdef __HIP__
+    c = ::__builtin_amdgcn_mbcnt_lo(static_cast<int>(x), add);
+    c = ::__builtin_amdgcn_mbcnt_hi(static_cast<int>(x >> 32), c);
+    #else
```

Comment:
is this formatted right?

Reply:
I think so. Have you confirmed that it flows down that path?

Reply:
We don't have a test for this. I was not told the failing build yet

Reply:
Flying blind unfortunately